ospf6d/ospf6_flood.h

```diff
@@ -58,5 +58,10 @@ extern void ospf6_install_lsa(struct ospf6_lsa *lsa);
 
 extern int config_write_ospf6_debug_flood(struct vty *vty);
 extern void install_element_ospf6_debug_flood(void);
+extern void ospf6_flood_interface(struct ospf6_neighbor *from,
+				  struct ospf6_lsa *lsa,
+				  struct ospf6_interface *oi);
+extern int ospf6_lsupdate_send_neighbor_now(struct ospf6_neighbor *on,
+					    struct ospf6_lsa *lsa);
```

Comment:
Why this function is not declared in ospf6d/ospf6_message.h while it is defined in ospf6d/ospf6_message.c (See below) ?

Please, put in coherence declaration in *.h and definition in *.c (choose ospf6_flood or ospf6_message, but not both).

Reply:
ospf6_message.h does not contain all the header files, similar to ospf6_lsupdate_send_neighbor and other declarations are in ospf6_flood.h file. 
I have moved the definition of ospf6_lsupdate_send_neigbor_now() to osp6_flod.c to make it coherent.

Reply:
I tried moving definition but ospf6 code need more than anticipated refactoring and cleanup of header dependencies. I think the author created flood.h file to keep all the declarations which are common between message.c and flood.c in same file. 
It would be good to take care of refactoring as separate item later.

Reply:
OK. that's fine for me